packages/istanbul-report-modern-html/src/filterButtons.js

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```

Comment:
maybe my knowledge is out of date, but can't we just `import React from 'react'`? The `React` object just needs to be in the namespace, to me this implies we're going to do something with it... again I could be wrong though.

Reply:
I used what I am used to.. and it works fine (flow requires "React" to be imported as * as).  but what you've suggested is what the official doc examples look like, so I'll change it to that